src/test/java/org/opensearch/sdk/TestSDKClient.java

```diff
@@ -98,6 +100,15 @@ public void testSDKRestClient() throws Exception {
         assertDoesNotThrow(() -> restClient.delete(new DeleteRequest(), ActionListener.wrap(r -> {}, e -> {})));
         assertDoesNotThrow(() -> restClient.search(new SearchRequest(), ActionListener.wrap(r -> {}, e -> {})));
         assertDoesNotThrow(() -> restClient.multiSearch(new MultiSearchRequest(), ActionListener.wrap(r -> {}, e -> {})));
+        assertDoesNotThrow(() -> restClient.performRequestAsync(new Request("", ""), new ResponseListener() {
```

Comment:
Let's make the dummy request more dummy? :P
```suggestion
        assertDoesNotThrow(() -> restClient.performRequestAsync(new Request("GET", "/"), new ResponseListener() {
```